synapse/models/orgs.py

```diff
@@ -485,6 +488,58 @@ def getModelDefs(self):
                     ('contest', ('ou:contest', {}), {
                         'doc': 'The contest that the person attended.',
                     }),
+                    ('preso', ('ou:preso', {}), {
+                        'doc': 'The presentation that the person attended.',
+                    }),
+                )),
+                ('ou:preso', {}, (
+
+                    ('organizer', ('ps:contact', {}), {
+                        'doc': 'Contact information for the primary organizer of the presentation.'}),
+
+                    ('sponsors', ('array', {'type': 'ps:contact'}), {
+                        'doc': 'An array of contacts which sponsored the presentation.'}),
+
+                    ('presenters', ('array', {'type': 'ps:contact'}), {
+                        'doc': 'An array of contacts which gave the presentation.'}),
+
+                    ('title', ('str', {'lower': True}), {
+                        'doc': 'The full name of the presentation.',
+                        'ex': 'Synapse 101 - 2021/06/22'}),
+
+                    ('desc', ('str', {'lower': True}), {
+                        'doc': 'A description of the presentation.',
+                        'disp': {'hint': 'text'}}),
+
+                    ('time', ('time', {}), {
+                        'doc': 'The presentation start time.'}),
+
+                    ('duration', ('duration', {}), {
+                        'doc': 'The duration of the presentation.'}),
+
+                    ('loc', ('loc', ()), {
+                        'doc': 'The geopolitical affiliation of the presentation.'}),
```

Comment:
doc wording? I feel like "geopolitical affiliation" is opaque. Compare (for example) with inet:ipv4:loc, "The geo-political location string for the IPv4." (slightly less opaque). Maybe "The geopolitical location string for where the presentation was given." with an ex: 'us.ma.boston' ?